From dbde4c033b7502b5a0651048239e6f505e16db02 Mon Sep 17 00:00:00 2001
From: Nicolas Chauvet <kwizart@gmail.com>
Date: Sun, 29 Jul 2018 22:12:39 +0200
Subject: [PATCH 1/2] Add neon fpu as compiler FLAGS

This is needed to compile neon asm

v2: use COMPILE_OPTIONS instead

Signed-off-by: Nicolas Chauvet <kwizart@gmail.com>
---
 xbmc/cores/VideoPlayer/VideoRenderers/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/xbmc/cores/VideoPlayer/VideoRenderers/CMakeLists.txt b/xbmc/cores/VideoPlayer/VideoRenderers/CMakeLists.txt
index ee9a10191f..43e9ac446b 100644
--- a/xbmc/cores/VideoPlayer/VideoRenderers/CMakeLists.txt
+++ b/xbmc/cores/VideoPlayer/VideoRenderers/CMakeLists.txt
@@ -56,6 +56,7 @@ endif()
 if(ARCH MATCHES arm)
   list(APPEND SOURCES yuv2rgb.neon.S)
   list(APPEND HEADERS yuv2rgb.neon.h)
+  set_source_files_properties(yuv2rgb.neon.S PROPERTIES COMPILE_OPTIONS -mfpu=neon)
 endif()
 
 core_add_library(videorenderers)
-- 
2.17.1

